src/woo_publications/publications/models.py

```diff
@@ -563,8 +563,8 @@ class PublicationIdentifier(models.Model):
     )
 
     class Meta:
-        verbose_name = _("kenmerk")
-        verbose_name_plural = _("kenmerken")
+        verbose_name = _("identifier")
```

Comment:
I think you want to use `pgettext` here to add a contextual marker so we can translate it differently to `kenmerk`, which is now not the case.

Reply:
I added a contextual marker but I don't know if the context given is any good.